src/sensors/bmi160sensor.cpp

```diff
@@ -131,6 +136,29 @@ void BMI160Sensor::motionLoop() {
     quaternion.set(-q[2], q[1], q[3], q[0]);
     quaternion *= sensorOffset;
 
+    if (SEND_ACCELERATION)
```

Comment:
Here and in all other occurrences, use preprocessor macro instead of operator to exclude unused code.
```suggestion
    #if SEND_ACCELERATION
```